python/lttoolbox.i

```diff
@@ -0,0 +1,50 @@
+%module lttoolbox
+
+%{
+#define SWIG_FILE_WITH_INIT
+#include <lttoolbox/fst_processor.h>
+#include <lttoolbox/lttoolbox_config.h>
+#include <lttoolbox/my_stdio.h>
+#include <lttoolbox/lt_locale.h>
+
+class FST: public FSTProcessor
+{
+public:
+  /**
+   * Reads from input_path and stores result at output_path
+   */  
+  void analyze(char *automorf_path, char *input_path, char *output_path);
+};
+
+
+void 
+FST::analyze(char *automorf_path, char *input_path, char *output_path)
```

Comment:
Is this necessary anymore with the new approach in apertium-python? Also, I think you didn't do `setDictionaryCaseMode(true);`  and `LtLocale::tryToSetLocale();` there?

Reply:
we do need to wrap `FSTProcessoranalysis` as it uses `FILE*`.